arithmetization/src/main/java/net/consensys/linea/zktracer/container/module/CountingOnlyModule.java

```diff
@@ -40,7 +40,7 @@ default int lineCount() {
     return counts().lineCount();
   }
 
-  default void addPrecompileLimit(final int count) {
+  default void addLimit(final int count) {
     Preconditions.checkArgument(count >= 0, "Must be positive");
```

Comment:
Rien à voir but the correct way to express `≥0` is "nonnegative", "positive means `>0`.